src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt

```diff
@@ -144,17 +151,25 @@ open class SipMessageHandler : AbstractVerticle() {
                 put("dst_port", dst.port)
                 dst.host?.let { put("dst_host", it) }
 
-                put("attributes", packet.attributes)
                 put("payload", message.headersMap())
+                put("attributes", mutableMapOf<String, Any>())
             }
 
-            vertx.eventBus().request<Boolean>(Routes.sip_message_udf, udf, USE_LOCAL_CODEC) { asr ->
-                if (asr.failed()) {
-                    logger.info("SipMessageHandler `callUserDefinedFunction()` failed.", asr.cause())
+            GlobalScope.launch(vertx.dispatcher()) {
+                val result = withTimeoutOrNull(executeUdfTimeout) {
```

Comment:
What do you think about using `*Await` functions? 
```kotlin
val result = withTimeoutOrNull(executeUdfTimeout) {
    vertx.eventBus().requestAwait<Boolean>(Routes.sip_message_udf, udf, USE_LOCAL_CODEC)
}
```
For me it's better to read and understand.

Reply:
agree